src/hammer-tech/stackup.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+#  stackup.py
+#  Data structures to represent technology stackup options
+#
+#  See LICENSE for licence details.
+
+from enum import Enum
+from typing import Any, Callable, Iterable, List, NamedTuple, Optional, Tuple, Dict
+from hammer_utils import reverse_dict
+
+class RoutingDirection(Enum):
```

Comment:
Random thought that could be filed as issue, do older technologies have bidirectional metals? Would those be labeled as a fourth category or would treating them as redistribution be ok?

Reply:
This is referring to the preferred routing direction rather than a DRC rule. Even older techs still do VHVHV... because having two same-direction metals back-to-back is bad for routing.